common/src/main/resources/startup.properties

```diff
@@ -83,6 +83,7 @@
 
 ##### Workflow Job Execution Completion listeners #####
 *.workflow.execution.listeners=
+#org.apache.falcon.service.LogMoverService
```

Comment:
Relate the 2 properties with each other through comments so that the users know that both should be switched together.